src/Microsoft.AspNetCore/WebHost.cs

```diff
@@ -173,8 +173,9 @@ public static IWebHostBuilder CreateDefaultBuilder(string[] args)
                         config.AddCommandLine(args);
                     }
                 })
-                .ConfigureLogging(logging =>
+                .ConfigureLogging((hostingContext, logging) =>
                 {
+                    logging.UseConfiguration(hostingContext.Configuration.GetSection("Logging"));
```

Comment:
@BrennanConroy can we get a basic test for this added so we can get this into Preview1?